packages/editor-ui/src/components/ParameterInputFull.vue

```diff
@@ -18,26 +18,57 @@
 			/>
 		</template>
 		<template>
-			<DraggableTarget type="mapping" :disabled="parameter.noDataExpression || isReadOnly" :sticky="true" :stickyOffset="4" @drop="onDrop">
-				<template v-slot="{ droppable, activeDrop }">
-					<parameter-input
-						ref="param"
-						:parameter="parameter"
-						:value="value"
-						:displayOptions="displayOptions"
-						:path="path"
-						:isReadOnly="isReadOnly"
-						:droppable="droppable"
-						:activeDrop="activeDrop"
-						:forceShowExpression="forceShowExpression"
-						@valueChanged="valueChanged"
-						@focus="onFocus"
-						@blur="onBlur"
-						inputSize="small" />
-				</template>
-			</DraggableTarget>
+			<div
```

Comment:
would be cleaner if this was part of `ParameterInput`... 
```
 <ParameterInputFull>
      <ParameterInput>
         <ResourceLoader v-if="type === 'resourceLoader">
```

Reply:
also would allow it to be reused in credentials ParameterInputExpanded

Reply:
That was my initial way to to but the problem here is that whole `ParameterInput` component is wrapped inside `DraggableTarget` so adding resource locator inside it would result in making both parts of it (mode selector and input fileds) drop-able which looks really wierd.
Alternative to this is to make a completely new component (`ResourceLocator`) and render it inside `ParameterInputFull` instead of `DraggableTarget` when `type === resourceLocator`. Thoughts?

Reply:
I would say just disable draggable target if it's resource loader.. then add it within the lower level component

Reply:
Agree. Refactored that in the latest commit.